locales/zh-TW.json

```
@@ -1,136 +1,136 @@
+  "myRecords:confirmDeleteRecord": "確定要刪除此記錄？",
+  "recordReport:pageTitle": "CPD 記錄",
+  "recordReport:cpdPeriod": "CPD 期間：",
+  "shared:creditHoursText": "學分小時數",
+  "addRecord:creditMinutes": "學分分鐘數",
+  "manageTargets:currentTargetPeriod": "開始日：",
+  "shared:dateCompletedText": "完成日期",
+  "pendingRecords:dateIssued": "發行日期",
+  "shared:dateRangeText": "日期範圍",
+  "manageTargets:day": "日期",
+  "adminItems:delete": "刪除",
+  "myRecords:deleteItem": "刪除 {itemName}",
+  "pendingRecords:dismissRecord": "關閉記錄",
+  "pendingRecords:dismissedRecords": "已關閉記錄",
+  "adminItems:edit": "編輯",
+  "addRecord:editCPD": "編輯 CPD 記錄",
+  "cpdAdmin:editMethod": "編輯方法",
+  "cpdAdmin:editQuestion": "編輯問題",
+  "cpdAdmin:editSubject": "編輯學科",
+  "manageTargets:editSubjectTarget": "編輯 {subject} 學科目標",
+  "manageTargets:editTargetPeriod": "編輯目標期間",
+  "myTeam:employeeName": "員工名稱",
+  "addRecord:enterCreditHours": "輸入學分小時數",
+  "addRecord:enterCreditMinutes": "輸入學分分鐘數",
+  "readOnlyRecord:evidence": "證明",
+  "myRecords:exportRecords": "匯出記錄",
+  "shared:filterSelect:ariaFilterRecordsBy": "依據 {choice} 篩選記錄",
+  "widget:goToCpd": "移至我的 CPD 儀表板",
+  "shared:gradeText": "成績",
+  "cpdAdminJobList:hasDefaults": "有預設",
+  "recordReport:creditHoursHeading": "學分小時數：",
+  "recordReport:dateCompletedHeading": "完成日期：",
+  "recordReport:evidenceHeading": "證明：",
+  "recordReport:gradeHeading": "成績：",
+  "recordReport:methodHeading": "方法：",
+  "recordReport:name": "名稱：",
+  "recordReport:recordNameHeading": "記錄名稱：",
+  "recordReport:recordTypeHeading": "類型：",
+  "recordReport:subjectHeading": "學科：",
+  "shared:hideSearchOptionsText": "隱藏搜尋選項",
+  "manageTargets:hours": "小時",
+  "shared:hoursProgress": "{numerator}/{denominator} 小時",
+  "cpdAdminJobList:jobTitle": "職稱",
+  "cpdAdmin:jobTitleTargets": "職稱目標",
+  "manageTargets:manageJobTargets": "管理職稱目標：{title}",
+  "cpdAdmin:manageMethods": "方法",
+  "manageTargets:managePersonalTargets": "管理個人目標",
+  "cpdAdmin:manageQuestions": "問題",
+  "cpdAdmin:manageSubjects": "學科",
+  "shared:methodText": "方法",
+  "cpdAdmin:methodName": "方法名稱：",
+  "cpdAdmin:methodNamePlaceholder": "輸入方法名稱",
+  "manageTargets:minutes": "分鐘",
+  "manageTargets:month": "月",
+  "cpd:myCPDTabHeader": "我的 CPD 記錄",
+  "shared:overallProgress:myTargetsProgressText": "我的目標進度",
+  "cpd:myTeamCpdTabHeader": "我的團隊 CPD",
+  "recordReport:na": "不適用",
+  "shared:nameText": "名稱",
+  "shared:pageSelect:ariaNextPageText": "下一頁",
+  "myRecords:no": "否",
+  "readOnlyRecord:noEvidence": "沒有附加到此記錄的證明。",
+  "readOnlyRecord:noGrade": "沒有已記錄的成績。",
+  "shared:noResultsFoundText": "找不到結果",
+  "pendingRecords:options": "選項",
+  "shared:overallProgress:combinedText:recordDuration": "您已完成達成整體專業發展目標所需的 {denominator} 小時中的 {numerator} 小時。",
+  "shared:overallProgressText": "整體進度",
+  "recordReport:overallProgressSummary": "整體進度摘要",
+  "shared:overallProgress:splitText:RecordDuration": "在您目標的 {total} 小時中，您已完成 {structured} 小時的結構性和 {unstructured} 小時的非結構性專業發展。",
```

We should probably maintain consistent casing, so something like 
```suggestion
  "shared:overallProgress:splitText:recordDuration": "在您目標的 {total} 小時中，您已完成 {structured} 小時的結構性和 {unstructured} 小時的非結構性專業發展。",
```
?

---

And you'd probably need to change RecordCount too.

---

Updated all the attributes to consistent casing.